### ISSUE TITLE
Using devtools::install_github additional arguments

### ISSUE THREAD

**Issue by grayskripko** [Azure/doAzureParallel#102]
[catboost R installation](https://tech.yandex.com/catboost/doc/dg/concepts/r-installation-docpage/) requires ```subdir ``` argument. How can I do this?

**Comment by brnleehng** [comment]
There's a larger discussion about how we should be managing package installation. I'll definitely add this to our discussion.

https://github.com/Azure/doAzureParallel/issues/51

One workaround:
1) Writing a R script that installs the catboost R package
2) Create a container
3) Upload the catboost R package
4) Reference it as a Azure resource file
5) Adding a command line to run your catboost R package script in your configuration file will do the trick
```json
commandLine": [
    "Rscript $AZ_BATCH_NODE_STARTUP_DIR/wd/catboost.R"
  ]
```

First link is about running command lines on the node when it gets boot up the first time:
https://github.com/Azure/doAzureParallel/blob/master/docs/30-customize-cluster.md

Second link is referencing resource files:
https://github.com/Azure/doAzureParallel/blob/master/samples/resource_files/sas_resource_files_example.R

Let me know if you have any issues :)

**Comment by paselem** [comment]
I was thinking about this earlier too. We need to document which environment variables are available for users to leverage when using the command line experience.

**Comment by grayskripko** [comment]
Thanks Brian.
I just used another one expression in cluster.json commandLine block 
```Rscript -e 'args <- commandArgs(TRUE)' -e 'devtools::install_github(args[1], subdir = args[2])' catboost/catboost catboost/R-package```

**Comment by paselem** [comment]
This should now be solved with doAzureParallel v0.6.0 where all of your packages can/should be defined in the docker image.
